src/BenchmarkDotNet/Running/BenchmarkRunner.cs

```diff
@@ -369,6 +369,19 @@ private static BenchmarkReport RunCore(Benchmark benchmark, BenchmarkId benchmar
 
                 executeResults.Add(executeResult);
 
+                var errors = executeResults.SelectMany(r => r.Data)
+                    .Union(executeResults.SelectMany(r => r.ExtraOutput))
+                    .Where(line => line.Contains(ValidationHelper.ConsoleErrorPrefix))
```

Comment:
nit: if this is a prefix, then maybe `StartsWith` would be better than `Contains`?